CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,15 @@
+# HOW TO CONTRIBUTE
+
+## "Our World Is One Country"
+
+Globe&Citizen, as a company and service, is based on the premise that people are _people_ all around the world and that all humans display the unique capacity to cooperate flexibly in groups. Participation requires, above all else, a willingness to work with other developers who are different from yourself respectfully, openly, in good faith.
+
+## Steps to Contributing
+
+1. Fork a local copy of the code base and get to know it.
```

Comment:
Why you say fork 

I think you want to say clone. 

Fork is used when you want to create a new version of a project with new gooal

Reply:
I was thinking this would be for members of the wider community who are not yet connected formally to the organization. People who want to start participating but are not yet member of Globe and Citizen. Primarily so that when they push, they can get it to their remote Github page and then use the "pull request" functionality. If they clone the repo, my understanding is that they won't be able to push.

Reply:
OK i get it

Reply:
@hermannleboss, is that ok to you? May we merge?

If yes, please "Resolve conversation".